pcsx2/gui/Panels/MemoryCardListPanel.cpp

```diff
@@ -122,11 +122,11 @@ bool EnumerateMemoryCard(McdSlotItem& dest, const wxFileName& filename, const wx
 		return false;
 	}
 
-	dest.IsPresent = true;
-	dest.Filename = filename;
-	if (filename.GetFullPath() == (basePath + filename.GetFullName()).GetFullPath())
-		dest.Filename = filename.GetFullName();
-
+	dest.IsPresent		= true;
+	dest.Filename		= filename.GetFullPath().ToStdString();
+	if( filename.GetFullPath() == (basePath+filename.GetFullName()).GetFullPath() )
```

Comment:
I've seen you replacing `+` of paths with `+` as well as `Path::Combine`.  I don't know which of these is the original functionality but it can't be both

Reply:
> 
> 
> BTW why use `Path::Combine` over `/` to replace `+`?
> 
> Also can you go over why you rewrote `AppUserMode` and what you changed? Was the old `AppUserMode` broken in some way? Or did you have some other reason for rewriting it?

Part of it was to clear WX from code it didn't need to be in. Part of it was for the sake of ghc / std filesystems use. AppUserMode is where our folder creation happens. Part of it was AppUserMode albeit functional was a mess that I took it upon myself to clean up. The current iteration accomplishes the same Goals using filesystem and a much easier and cleaner logic to understand.